README.md

```diff
@@ -58,7 +58,7 @@ c.NotebookApp.contents_manager_class = BookstoreContentsArchiver
 
 # All Bookstore settings are centralized on one config object so you don't have to configure it for each class
 c.BookstoreSettings.workspace_prefix = "/workspace/kylek/notebooks"
-c.BookstoreSettings.published_prefix = "/published/kylek/notebooks"
+c.BookstoreSettings.publish_prefix = "/published/kylek/notebooks"
```

Comment:
Please don't change this one, it's a major version change and I've already started using it elsewhere. It's only to reflect the prefix on the bucket.

Reply:
Sounds good, as I mention below, I was trying to fulfill what I thought was your request around the names. I’m not attached to this in any way.